client/src/main.rs

```diff
@@ -269,6 +269,11 @@ enum Command {
         #[clap(value_enum)]
         shell: clap_complete::Shell,
     },
+
+    ListCandidates {
```

Comment:
Let's give this a doc comment. I'd have no idea what "candidates" are as a user of innernet.